helpers/getPartyCompFromRoomData.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var _ = require('lodash');
+var pluralise = require('../lib/pluralise');
+
+module.exports = function(dust) {
+
+  /*
+  * @description Get the combined party composition of all the rooms
+  * @param {string} rooms rooms array to loop over
+  */
+
+  dust.helpers._getPartyCompFromRoomData = function(chunk, context, bodies, params) {
+    params = params || {};
+    var adults = 0;
+    var children = 0;
+    _.forEach(params.rooms, function(room) {
```

Comment:
1. Why not use Array.prototype.forEach in this case. You'll have the same result and won't have to include a library.
2. JSDoc is wrong, there is no 'rooms' param, and the other four are missing. Also you could skip the new line between the function and the jsDoc but that is up to you
3. If the params parameter is missing and the value is {} wouldn't that throw an error when you try to iterate trough rooms, when that might not exist?

Reply:
1. We were already using the library, so we can just use it here. I guess we can just use the native array version, but it helps resolve in the case of `params.rooms` being undefined (as per your 3rd point).
2. Changed the JSDoc, removed the new line and the params.